hiera.yaml

```diff
@@ -2,15 +2,21 @@
 version: 4
 datadir: data
 hierarchy:
+  - name: "OS + Release"
+    backend: "yaml"
+    path: "os/%{facts.operatingsystem}-%{facts.operatingsystemmajrelease}"
+  - name: "OS"
+    backend: "yaml"
+    path: "os/%{facts.operatingsystem}"
   - name: "OSFamily + Release"
     backend: "yaml"
-    path: "os/%{facts.osfamily}-%{facts.operatingsystemmajrelease}"
+    path: "os/family/%{facts.osfamily}-%{facts.operatingsystemmajrelease}"
```

Comment:
This doesn't make sense. Families don't have releases.

Reply:
@trevor-vaughan I can't speak for whoever originally designed this hierarchy, but it makes perfect sense to me: we routinely speak of "EL7" and "EL6" with the understanding that they are  `:operatingsystem`-independent releases that are consistent across the entire `RedHat` OS family (e.g., CentOS-7 and RedHat-7).